### PR TITLE
Give seller for active board

### PR DESCRIPTION
**Pull request by marcmerlin**
As requested in https://github.com/hzeller/rpi-rgb-led-matrix/issues/797

### CODE REVIEW

**Comment by marcmerlin** [comment]
@hzeller , ping? :)

File: adapter/README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ with the wiring when using the `rpi-rgb-led-matrix` code.
      buffer the outputs using the ...
    * [Active-3](./active-3) board. Supports three parallel chains with active buffering
      and 3.3V -> 5V level shifting for best reliability. Requires SMD soldering.
+     As another option you can buy it from these locations:
+     * https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-raspberry-pi/
+       ($3/board, but fairly long and/or expensive shipping from HKG)
```
Comment (hzeller) [inline adapter/README.md]: Please add a "Not affiliated" disclaimer, as I don't want to give support for their product.

Comment (marcmerlin) [reply adapter/README.md]: Fair point, done.

**Comment by marcmerlin** [comment]
Hopefully https://github.com/hzeller/rpi-rgb-led-matrix/pull/799/files looks good now?

**Comment by hzeller** [comment]
Thanks.

**Pull request merged**